lib/contracts/index.ts

```diff
@@ -65,4 +66,17 @@ export const contracts: ContractDefinition[] = [
 	viewAllTransformers,
 	viewAllViews,
 	viewScheduledActions,
+
+	relationshipAnyIsCreatorOfAny,
+	relationshipContactHasBackupOwnerUser,
+	relationshipContactIsAttachedToUser,
+	relationshipContactIsOwnedByUser,
+	relationshipCreateIsAttachedToAny,
+	relationshipExecuteExecutesActionRequest,
+	relationshipLoopOwnsTransformer,
+	relationshipTaskGeneratedAny,
+	relationshipTransformerGeneratedTask,
+	relationshipTransformerWorkerOwnsTask,
+	relationshipUpdateIsAttachedToAny,
+	relationshipUserHasAttachedContactContact,
```

Comment:
These are duplicates. Also, `relationshipUserHasAttrachedContactContact` should be `relationshipUserHasAttachedContact` no?